pom.xml

```diff
@@ -94,7 +94,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <scope>test</scope>
+            <version>2.35</version>
```

Comment:
The version is defined from the BOM which is why it wasn't explicitly defined here. Is there a good reason to add it back it? Defeats the purpose of using the BOM, doesn't it?